environment-api/src/main/java/com/sequenceiq/environment/api/v1/environment/model/EnvironmentNetworkGcpParams.java

```diff
@@ -45,15 +45,15 @@ public void setSharedProjectId(String sharedProjectId) {
         this.sharedProjectId = sharedProjectId;
     }
 
-    public Boolean getNoPublicIp() {
+    public Boolean isNoPublicIp() {
         return noPublicIp;
     }
 
     public void setNoPublicIp(Boolean noPublicIp) {
         this.noPublicIp = noPublicIp;
     }
 
-    public Boolean getNoFirewallRules() {
+    public Boolean isNoFirewallRules() {
```

Comment:
Like above, the `getX()` naming is the appropriate one due to the usage of a reference type.